torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -161,6 +161,48 @@ static inline at::optional<int64_t> parse_as_integer(const std::string& s) {
   return (*str_end == 0) ? at::optional<int64_t>(ans) : at::nullopt;
 }
 
+/*
+Parse default value of IntList declared at native_functions.yaml
+
+There are two kinds of default values:
+1. IntList[2] x=1 (where size=2, value={1,1}
+2. IntList x={1,2,3} (where size=3, value={1,2,3}, note that there cannot be space after comma since native_parse.py uses ', ' to split args)
+*/
+static inline std::vector<int64_t> parse_intlist_args(const std::string& s, int64_t size) {
+  if (s[0] != '{') {
+    return std::vector<int64_t>(size, std::stoi(s));
+  }
+
+  size_t n = s.size();
+  // since already checked left brace '{' above, here only checks right brace '}'
+  AT_CHECK(s[n - 1] == '}', "Default value of IntList is missing right brace '}', found ", s[n - 1]);
+
+  auto args = std::vector<int64_t>();
+  // for case IntList x={}, return an empty vector
+  if (s.size() == 2) {
+    return args;
+  }
+
+  int64_t x = 0, sign = 1;
+  for (size_t i = 1; i < n - 1; i++) {
```

Comment:
why doesn't this just do what parse int64 does -- atol?  Find the next command or if none exists parse to the end.

Reply:
Yes, I can use atol